src/fetchtastic/client_app_config.py

```
@@ -214,6 +217,32 @@ def normalize_client_app_config(config: dict[str, Any]) -> dict[str, Any]:
         config["CHECK_DESKTOP_PRERELEASES"] = False
+
+    config["CHECK_APP_SNAPSHOTS"] = coerce_bool(
+        config.get("CHECK_APP_SNAPSHOTS", DEFAULT_CHECK_APP_SNAPSHOTS),
+        default=DEFAULT_CHECK_APP_SNAPSHOTS,
+    )
+    snapshots_need_apks = config["CHECK_APP_SNAPSHOTS"]
+    if snapshots_need_apks:
+        if not coerce_bool(config.get("SAVE_CLIENT_APPS", False)):
+            config["CHECK_APP_SNAPSHOTS"] = False
+        elif not config.get("SELECTED_APP_ASSETS"):
+            config["CHECK_APP_SNAPSHOTS"] = False
```

Ruff (0.15.20)

[warning] Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)